eureka-service/src/main/resources/application.yml

```
@@ -10,6 +10,6 @@ eureka:
     register-with-eureka: false
     fetch-registry: false
     service-url:
-      defaultZone: http://localhost:19090/eureka/
+      defaultZone: http://eureka-service:19090/eureka/
   instance:
-    hostname: localhost
\ No newline at end of file
+    hostname: eureka-service
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)